Messenger/AppDelegate.mm

```diff
@@ -173,6 +168,27 @@ - (void)applicationDidBecomeActive:(NSNotification *)notification {
 }
 
 
+#pragma mark - Javascript events
+
+- (id)evaluateWebScript:(NSString *)script {
```

Comment:
If you think we should do this, we should refactor _all_ code to use such a convenience function, and it should probably come in two flavors, as suggested here: https://github.com/rsms/fb-mac-messenger/pull/53#discussion_r28293977